### DISCUSSION TITLE
Compiling Steps

### DISCUSSION THREAD

**Discussion by rubynt** [Difegue/Stylophone#48] [Q&A] [answered]
Because I dont know how to use VS/Windows and I dont use C# can we get steps to how to compile it?

**Comment by Difegue** [comment, marked as answer]
I'd very much recommend you install VS with the universal windows workflows in any case, since the UWP portion of the app will probably only compile with msbuild and the associated Win10 SDK.  

You can then look at the CI workflow: https://github.com/Difegue/Stylophone/blob/dev/.github/workflows/build-and-test.yml for example CLI commands.  (Although once again it's probably going to be miles easier to just build the .sln in VS directly)
